package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "eslint . --report-unused-disable-directives --max-warnings 0"
```

Comment:
Is there a reason this rule is not in the config file?

Reply:
Not other than me doing this in a rush. Will fix. Good find.